src/App.vue

```diff
@@ -1,25 +1,32 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
```

Comment:
Let's leave HelloWorld in here for now as the landing page. That way, we can see how the Header acts on a page, instead of just seeing it in isolation. For example, if you do this:

`
<div>
  <sticky-header/>
  <HelloWorld msg="Welcome to Your Vue.js App"/>
  <footer/>
</div>
`
you'll see that the header is transparent, so its content interferes with the main page content.